Makefile

```diff
@@ -362,11 +363,11 @@ config.mk:
 xlicense.h: LICENSE
 	$(TEXTCONV) $< > $@
 
-ALL_SRCS:=$(wildcard filter/*.cpp rss/*.cpp src/*.cpp test/*.cpp)
-ALL_HDRS:=$(wildcard filter/*.h rss/*.h test/*.h 3rd-party/*.hpp) $(STFLHDRS) xlicense.h
+ALL_SRCS:=$(wildcard filter/*.cpp rss/*.cpp src/*.cpp test/*.cpp test/test-helpers/*.cpp)
+ALL_HDRS:=$(wildcard filter/*.h rss/*.h test/test-helpers/*.h 3rd-party/*.hpp) $(STFLHDRS) xlicense.h
 depslist: $(ALL_SRCS) $(ALL_HDRS)
 	> mk/mk.deps
-	for dir in filter rss src test ; do \
+	for dir in filter rss src test test/test-helpers ; do \
 		for file in $$dir/*.cpp ; do \
```

Comment:
Yes! I made the change. That changed the ordering in mk/mk.deps. I hope Make's `wildcard` is deterministic, and will produce the same new ordering on CI as well.

Reply:
…and [it didn't produce the same output](https://cirrus-ci.com/task/4760145972428800?command=main#L152).

Reply:
I tried `$(shell ls -1 <patterns>)` instead of `$(wildcard <patterns>)`, and that seem to work [much more reliably](https://cirrus-ci.com/task/6268802224619520). I'm not sure if it'll keep working on other distros or OSes, though. Let's take a risk :)

Thanks for the suggestion, @dennisschagt! I'll merge this when the builds finished.